src/Nordea/Components/Table.elm

```diff
@@ -57,8 +61,8 @@ view attrs children =
 
 {-| Groups the header content in a table
 -}
-theadWithVariant : List (Attribute msg) -> List (Html msg) -> Variant -> Html msg
-theadWithVariant attrs children variant =
+theadWithVariant : Variant -> List (Attribute msg) -> List (Html msg) -> Html msg
+theadWithVariant variant attrs =
```

Comment:
I guess one thing to consider is if you want to write code like:
```
ExtraSmall
|> theadWithVariant [] [ Html.text ""]
```
This will not work, but I'm not certain if its nice

Reply:
Discussed this with @fredrikloberg now and I think having `Variant` as the first argument is fine for now. 

We are using the [builder pattern](https://sporto.github.io/elm-patterns/basic/builder-pattern.html) for a lot of other components and there it makes sense to pass in `Variant` as the last argument, but it is not the case for the functions in the Table component.

If we were to improve the Table component in the future and wanted to have optional parameters, we could then either go for the builder pattern or just an [argument list](https://sporto.github.io/elm-patterns/basic/arguments-list.html) which might be neater (and is already what is used in i.e. the Elm Html package).